setup.py

```diff
@@ -17,6 +17,10 @@
 import re
 from setuptools import setup, find_packages
 
+cur_dir = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(cur_dir, 'README.md'), 'rb') as f:
+    long_description = f.read().decode('utf-8')
```

Comment:
Filter images label in Readme.md cause images cannot be displayed in the description page.